src/libanalyze/mesontree.cpp

```diff
@@ -186,20 +186,30 @@ std::shared_ptr<Node> MesonTree::parseFile(const std::filesystem::path &path) {
   return rootNode;
 }
 
+std::shared_ptr<Node> MesonTree::parseRootFile() {
+  const auto &rootFile = this->root / "meson.build";
+  if (this->asts.contains(rootFile)) {
+    return this->asts.at(rootFile).back();
```

Comment:
I don't know if this would cause a bug when calling `.back()`.
Is the root file supposed to be the last element of `this->asts.at(rootFile)` in this stage?